dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -77,16 +97,96 @@ class Catalog extends React.Component<ICatalogProps, ICatalogState> {
           />
         </PageHeader>
         <LoadingWrapper loaded={!isFetching}>
-          <CardGrid>{chartItems}</CardGrid>
+          <div className="row">
+            {csvs.length > 0 && (
+              <div className="col-2">
+                <div className="margin-b-normal">
+                  <span>
+                    <b>Type:</b>
+                  </span>
+                </div>
+                <div>
+                  <label className="checkbox" key="listcharts">
+                    <input type="checkbox" checked={listCharts} onChange={this.toggleListCharts} />
+                    <span>Charts</span>
+                  </label>
+                </div>
+                <div>
+                  <label className="checkbox" key="listoperators">
+                    <input
+                      type="checkbox"
+                      checked={listOperators}
+                      onChange={this.toggleListOperators}
+                    />
+                    <span>Operator Instances</span>
```

Comment:
We're using the technical name "operator instances" here, but I wonder if it'd be better from a user pov to just filter by "operators" (vs charts)?

Reply:
Right, I used Operators first but I thougth it could be confusing with the operators view. In any case you are probably right.